source/common/quic/envoy_quic_stream.h

```diff
@@ -102,6 +107,21 @@ class EnvoyQuicStream : public virtual Http::StreamEncoder,
     filter_manager_connection_.updateBytesBuffered(old_buffered_bytes, new_buffered_bytes);
   }
 
+  Http::HeaderUtility::HeaderValidationResult
+  validateHeader(const std::string& header_name, absl::string_view header_value) override {
+    bool override_stream_error_on_invalid_http_message =
+        http3_options_.has_override_stream_error_on_invalid_http_message() &&
```

Comment:
Given we're cloning this, what do you think of just always setting it in initializeAndValidate?